### PR TITLE
google-apps-script May 20, 2019 update

### PR DESCRIPTION
**Pull request by PopGoesTheWza**
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://developers.google.com/apps-script/>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

@grant 

## May 20, 2019

- The Gmail service has been updated with the GmailMessage.getHeader(name) method that can retrieve a RFC 2822 header from a message.
- The Optimization service has been updated with the following batch methods:
  - LinearOptimizationEngine.addContraints(lowerBounds, upperBounds, variableNames, coefficients)
  - LinearOptimizationEngine.addVariables(names, lowerBounds, upperBounds, types, objectiveCoeffients)

### CODE REVIEW

File: types/google-apps-script/google-apps-script.gmail.d.ts [hunk 1]
```diff
@@ -154,58 +154,58 @@ declare namespace GoogleAppsScript {
...
-      /**
-       * The name of the sender of the email (default: the user's name).
-       */
-      name?: string;
-      /**
-       * An email address to use as the default reply-to address (default: the user's email address).
-       */
-      replyTo?: string;
-    }
+        /**
```
Comment (grant) [inline types/google-apps-script/google-apps-script.gmail.d.ts]: Undo this.

File: types/google-apps-script/google-apps-script.gmail.d.ts [hunk 2]
```diff
@@ -154,58 +154,58 @@ declare namespace GoogleAppsScript {
...
+        replyTo?: string;
+      }
 
-    /**
-     * Options for a Gmail Attachments.
-     */
-    export type GmailAttachmentOptions = {
       /**
-       * If the returned array of Blob attachments should include inline images.
+       * Options for a Gmail Attachments.
```
Comment (grant) [inline types/google-apps-script/google-apps-script.gmail.d.ts]: Please format this file.

**Review by grant — CHANGES_REQUESTED** [review]
Thanks.
Please format the file.

**Comment by PopGoesTheWza** [comment]
@grant it should be good now

**Review by grant — APPROVED** [review]

**Review by grant — APPROVED** [review]

**Pull request merged**